src/Message/PaymentIntents/AuthorizeRequest.php

```diff
@@ -127,26 +127,6 @@ public function getConfirm()
         return $this->getParameter('confirm');
     }
 
-    /**
-     * Set the confirm parameter.
-     *
-     * @param $value
-     */
-    public function setOffSession($value)
```

Comment:
Why is this removed?

Reply:
They are dupplicated since  https://github.com/thephpleague/omnipay-stripe/pull/177 and the build failed

(see https://github.com/thephpleague/omnipay-stripe/pull/177#issuecomment-643137388)

I already removed them in https://github.com/thephpleague/omnipay-stripe/pull/182 that you merged this morning. I'll rebase this PR